### DISCUSSION TITLE
after server restart, imagick not available to PHP7.4

### DISCUSSION THREAD

**Discussion by NFGman** [ImageMagick/ImageMagick#3810] [Help]
After a server (ubuntu server 20.04) reboot this morning, imagick is no longer available to my PHP7.4 installation.  It worked fine yesterday, it's been working fine for years.  I use it across a dozen websites and it's never been anything less than painless and effective.  And now, 'class 'imagick' not found' is all I can get.

I've removed and purged and installed php-imagick to no avail.  PHP doesn't throw any errors as if it fails to load the module, it doesn't seem to even try.  PHP's extension_dir is 2019xxx and imagick.so installed to 2020xxx, but copying this file to the 2019 dir has no effect.

phpinfo doesn't list imagick, php -m doesn't list it.  There are no errors in php -i.  running 'convert' from terminal works fine.  It's installed, but PHP can't see or use it.

I'm at my wits end, and two hours of searching for answers has gotten me nowhere.  I would appreciate any help.

**Comment by dlemstra** [comment]
You probably want to ask that question here instead: https://github.com/Imagick/imagick.
